specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```diff
@@ -442,7 +442,11 @@
         },
         "sparkEventsFolder": {
           "type": "string",
-          "description": "The Spark events folder"
+          "description": "The Spark events folder",
+          "x-ms-mutability": [
```

Comment:
Is it intentional that this field couldn't read?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability